### ISSUE TITLE
vue/v-bind-style not showing warnings on version 0.5

### ISSUE THREAD

**Issue by longtomjr** [nuxt/eslint-plugin-nuxt#76]
Good day. 
With the code 
```            
<error-message v-for="param in validationParameters" v-bind:error="!validationState[param]" :key="param" />
```
the warning and fix for v-bind-style does not trigger on version 0.5. After downgrading the error works again. Default nuxt config.

**Comment by clarkdo** [comment]
Can you check version of eslint, vue-eslint-parser and eslint-vue-plugin ?

**Comment by clarkdo** [comment]
BTW, are you using `@nuxtjs/eslint-config@1.x` ? If so, could you try to ugprade to v2 ? v2 has upgraded to eslint@6 and  eslint-vue-plugin@6.

**Comment by longtomjr** [comment]
Good day. Yes, updating eslint-config fixed the issue. Thanks

Kind Regards
